client-registry-jempi/docker-compose.combined.yml

```diff
@@ -91,8 +91,43 @@ services:
       kafka:
       default:
 
+  jempi-postgresql:
+    image: bitnami/postgresql:15.2.0
+    environment:
+      POSTGRESQL_USERNAME: ${POSTGRESQL_USERNAME}
+      POSTGRESQL_DATABASE: ${POSTGRESQL_DATABASE}
+      ALLOW_EMPTY_PASSWORD: "yes"
+    deploy:
+      mode: replicated
+      replicas: 1
+      resources:
+        limits:
+          cpus: ${JEMPI_POSTGRES_CPU_LIMIT}
+          memory: ${JEMPI_POSTGRES_MEMORY_LIMIT}
+        reservations:
+          cpus: ${JEMPI_POSTGRES_CPU_RESERVE}
+          memory: ${JEMPI_POSTGRES_MEMORY_RESERVE}
+    volumes:
```

Comment:
This would break in clustered mode, we should address it in the new sprint when we do JeMPI clustering. There should be a placement constraint on this service.